lib/medium-admin.php

```diff
@@ -1097,7 +1102,10 @@ private static function _handle_response($response) {
       $error_code = $response->get_error_code();
       error_log("WP ERROR: $message ($error_code)");
       if ($error_code == "http_request_failed" && strpos($message, "timed out") !== false) {
-        throw new Exception($message, -2); // our custom code for timeouts
+        throw new Exception($message, API_TIMEOUT);
+      }
+      if ($error_code == "http_request_failed" && strpos($message, "provisioned throughput for the table was exceeded") !== false) {
```

Comment:
Yup, that we do. And it does cause problems with migrations occasionally. Agreed, not such a great idea, but I think that this will lessen the impact on migrations that are happening as this is a frequent error.